### PR TITLE
Fix SQLITE_PATH instruction in backend readme

### PR DESCRIPTION
**Pull request by anthony2261**

### CODE REVIEW

File: frontend/src/components/Landing/Footer.tsx
Comment (RamiAwar) [inline frontend/src/components/Landing/Footer.tsx:1]: Remove this tho, I patched this morning

**Comment by RamiAwar** [comment]
LGTM

File: frontend/src/components/Landing/Footer.tsx
Comment (anthony2261) [reply frontend/src/components/Landing/Footer.tsx:1]: This was still there after your patch, that's how I found out hahah

Comment (anthony2261) [reply frontend/src/components/Landing/Footer.tsx:1]: image

**Review by RamiAwar — APPROVED** [review]

**Pull request merged**